O365-iOS-Microsoft-Graph-Connect-Swift/AuthenticationClass.swift

```diff
@@ -10,7 +10,7 @@ class AuthenticationClass {
     // Singleton class
     static let sharedInstance = AuthenticationClass()
 
-    var authenticationProvider = MSALPublicClientApplication.init()
+    var authenticationProvider = MSALPublicClientApplication()
```

Comment:
Doing this way we don't need to init empty MSALPublicClientApplication:

`var authenticationProvider: MSALPublicClientApplication!`

Reply:
agree, fixed!